src/plugin.json

```diff
@@ -54,14 +54,6 @@
       "role": "Viewer",
       "addToNav": true,
       "defaultNav": false
-    },
-    {
-      "type": "page",
-      "name": "Settings",
-      "path": "/a/%PLUGIN_ID%/settings",
-      "role": "Admin",
```

Comment:
@bryanhuhta  Isn't it weird that we scoped this page to admins only? After all they are user settings